src/main/java/TaskList.java

```diff
@@ -0,0 +1,109 @@
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> taskArr = taskArr = new ArrayList<>();
```

Comment:
This is interesting, I tried this on my jshell, and it works. Do you mind explaining how does it work? :)